jtalks-common-model/src/main/resources/org/jtalks/common/migrations/V8__New_Entities.sql

```diff
@@ -58,7 +58,7 @@ CREATE TABLE `GROUP_USER_REF` (
 
 CREATE TABLE `PROPERTIES` (
   `PROPERTY_ID` bigint(20) NOT NULL AUTO_INCREMENT,
-  `UUID` varchar(255) NOT NULL,
+  `UUID` varchar(100) NOT NULL,
   `NAME` varchar(255) NOT NULL,
```

Comment:
UUID is the indexed field in the PROPERTIES table while NAME is not indexed. Look:
```sql
CREATE TABLE `PROPERTIES` (
  `PROPERTY_ID` bigint(20) NOT NULL AUTO_INCREMENT,
  `UUID` varchar(100) NOT NULL,
  `NAME` varchar(255) NOT NULL,
  `VALUE` varchar(255) DEFAULT NULL,
  `CMP_ID` bigint(20) DEFAULT NULL,
  PRIMARY KEY (`PROPERTY_ID`),
  UNIQUE KEY `UUID` (`UUID`),
  KEY `FK_COMPONENT` (`CMP_ID`),
  CONSTRAINT `FK_COMPONENT` FOREIGN KEY (`CMP_ID`) REFERENCES `COMPONENTS` (`CMP_ID`)
);
```
From [MySQL official documentation](https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-conversion.html):
> InnoDB has a maximum index length of 767 bytes, so for utf8 or utf8mb4 columns, you can index a maximum of 255 or 191 characters, respectively